wiki/Beatmap_search/en.md

```diff
@@ -98,6 +99,7 @@ Beatmap metadata fields can be compared against specific values for more granula
 | `divisor` | The denominator of the [beat snap divisor](/wiki/Client/Beatmap_editor/Beat_snap_divisor) |
 | `submitted` | Beatmap creation/upload date |
 | `ranked` | Beatmap ranking/approval date |
+| `tag` | Specific [user tag](/wiki/Beatmap/Beatmap_tags#user-tags) (only supported in the built-in beatmap listing) |
```

Comment:
If the `Client (lazer)` section refers specifically to the song selection (and not the integrated beatmap listing), then this should be made more clear instead of adding this line.